include/ebpf_api.h

```diff
@@ -412,10 +412,10 @@ extern "C"
      * @brief Close an eBPF handle.
      *
      * @param[in] handle Handle to close.
-     * @retval ERROR_SUCCESS Handle was closed.
-     * @retval ERROR_INVALID_HANDLE Handle is not valid.
+     * @retval EBPF_SUCCESS Handle was closed.
+     * @retval EBPF_INVALID_OBJECT Handle is not valid.
      */
-    uint32_t
+    ebpf_result_t
```

Comment:
Just mentioning: handle based APIs will be removed (once all usage of handle based API is changed, tracked by #383 ). So we may not need to fix them.

Reply:
it shouldn't hurt though, right?

Reply:
Yes no problem in fixing it.